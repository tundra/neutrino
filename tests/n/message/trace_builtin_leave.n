# Copyright 2014 the Neutrino authors (see AUTHORS).
# Licensed under the Apache License, Version 2.0 (see LICENSE).

import $core;

def $fail_builtin() => (@core:Array.new(1))[100];
do $fail_builtin();
