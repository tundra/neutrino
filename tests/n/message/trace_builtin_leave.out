--- backtrace ---
- leave.out_of_bounds(100)
- [null][100]
- :fail_builtin()
Error: %<condition: Signal(escape)>
